CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.15.1 (July 2020)
+- Android 10 support
+- Update network libraries to more recent versions, OkHttp + Dav4Android
+- Bug fixes, including:
+  + Fix some authentication problems
```

Comment:
Could you mention those authentication problems in order to let the users with those problems know that are already fixed?